src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private final String color;
+
+    public Figure() {
+        this.color = ColorSupplier.getRandomColor();
```

Comment:
```suggestion
    public Figure(String color) {
        this.color = color;
```
not recommended to use random as default behavior